### DISCUSSION TITLE
Macro list-links-draggable has not emptyMessage

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#6450] [Ideas]
While in `$:/core/macros/list` it can be seen that `list-tagged-draggable` supports **emptyMessage**, but `list-links-draggable` does not!

It is good to be able to show an empty message, users can realize there is an area for drag and drop tiddlers.

**Comment by pmario** [comment]
>It is good to be able ...

Starts like a question, but the sentence doesn't have a questionmark. So what exactly is your idea? .. It looks like a "feature request" for me.

**Comment by kookma** [comment]
Hi Mario

* My question is: is there any reason `list-links-draggable` has not **emptyMessage** parameter but `list-tagged-draggable` has?
* I recommend to add **emptyMessage** parameter to `list-links-draggable`.

**Reply by Jermolene** [reply]
Hi @kookma that makes sense, do please feel free to make a ticket.

**Reply by pmario** [reply]
@kookma .. It is possible to convert a discussion into an issue. There is a button / link in the right sidebar.

**Reply by Jermolene** [reply]
In general, it's better to make a new issue that clearly expresses the proposal. It's time consuming to have to read through a long discussion thread to figure out the actual proposal. Of course, in this case this thread is quite short, but the OP would still need rewording to make sense as an issue.

**Reply by kookma** [reply]
I converted this discussion to issue and edited it to be self explaining! Hope this helps!

**Comment by kookma** [comment]
Thank you Jeremy! I will try to make a PR!
